app/controllers/v1/facilities/ccp_controller.rb

```diff
@@ -35,16 +32,41 @@ def specialties
       )
     end
 
-    render_json(PPMS::SpecialtySerializer, ppms_params, api_results)
+    render_json(PPMS::SpecialtySerializer, params, api_results)
   end
 
   private
 
+  def index_api_results
+    if Flipper.enabled?(:facility_locator_ppms_use_v1_client)
+      index_api_results_v1
+    else
+      index_api_results_v0
+    end
+  end
+
+  def index_api_results_v0
+    ppms_search.collect do |result|
```

Comment:
it looks like in the search v1 uses pagination, and here it's v0 using pagination?  Do I understand that correctly?  I'm surprised they don't align.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/7782-changes-to-ppms-queries-pt4-integration-with-feature-flag/lib/facilities/ppms/v1/response.rb#L51-L56

its one of the improvements for v1

Reply:
ah,  i thought this was adding `.paginate(pagination_params)` to the v0.  I see better what's happening.